pandas/tests/indexes/period/test_astype.py

```diff
@@ -0,0 +1,99 @@
+# -*- coding: utf-8 -*-
+
+import numpy as np
+import pytest
+
+import pandas as pd
+import pandas.util.testing as tm
+from pandas import NaT, Period, PeriodIndex, Int64Index, Index, period_range
+
+
+class TestPeriodIndexAsType(object):
```

Comment:
can you remove  the class. normally don't use a class unless you have 2 (and then I would question why we are not doing in sub-modules), unless very short

Reply:
I tend to agree on the why-is-a-class-needed bit.  I'd also like to keep symmetry with the test_astype modules in tests.indexes.datetimes and tests.indexes.timedeltas.  In an upcoming pass I'll de-class all three at the same time.